rerun-failed-run/rerun-failed-run.yml

```
@@ -0,0 +1,73 @@
+name: Retrigger failed run action
+
+on:
+  workflow_dispatch:
+    inputs:
+      run_id:
+        description: The id of the run which we want to retrigger
+        required: true
+      error_message:
+        description: Custom error message to look for in the logs.
+        required: true
+      repository:
+        description: Github repository
+        required: true
+jobs:
+  rerun:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Import Secrets
+        id: secrets
+        uses: hashicorp/vault-action@v2.7.4
```

the latest version is [2.8.0](https://github.com/hashicorp/vault-action/releases):
```suggestion
        uses: hashicorp/vault-action@v2.8.0
```